Documentation/documentation.org

```diff
@@ -0,0 +1,259 @@
+#+title: Polyclot 0.0.1 documentation
+#+author: Daniel <jackdaniel> Kochmański
+#+email: daniel@turtleware.eu
+#+date: [2019-11-04 pon]
+
+* Overview
+
+Polyclot is a tool to draw interactive charts in CLIM. Purpose of this
+document is to provide information on how to plot data and how to
+create new types of charts. Tool may be used by other CLIM
+applications as a library or as a standalone utility to render charts.
+
+* Developer guide
+** Coding conventions
+- class names are enclosed in brackets, i.e ~<data-frame>~
+- type names are enclosed in percentage characters, i.e ~%index%~
+- anaphoric macro variable names shoud start with dash, i.e ~^it~
```

Comment:
Do you mean caret (instead of dash), or is it supposed to be ~-it~?

Reply:
I do mean carret, thanks (fixed)